editor/components/post-taxonomies/flat-term-selector.js

```diff
@@ -171,7 +171,6 @@ class FlatTermSelector extends Component {
 
 		return (
 			<div className="editor-post-taxonomies__flat-terms-selector">
```

Comment:
Do we need the wrapper anymore?

I see there's a lingering style for it, but I'm wondering if we need that either:

https://github.com/WordPress/gutenberg/blob/adb821746d9803ae304c57a750b824e08cdcacef/editor/components/post-taxonomies/style.scss#L1-L4

Why doesn't `.components-panel__body-title` add its own bottom margin rather than rely on children to add this themselves?